src/br/com/codar/maquinacafe/Cafeteira.java

```diff
@@ -0,0 +1,45 @@
+package br.com.codar.maquinacafe;
+
+import java.util.Scanner;
+
+public class Cafeteira {
+
+	public static void main(String[] args) {
+
+		Pedido pedido = new Pedido();
+		Credito credito = new Credito();
+		int iniciarEscolha;
+
+		Scanner pedidoInicial = new Scanner(System.in);
+		System.out.println("Ol�, escolha uma das op��es abaixo\n" + "1 - Adicionar cr�ditos\n" + "2 - Realizar pedido");
```

Comment:
Encapsular essas chamadas de System.out pode ser uma boa ideia. Se em algum momento for necessário refatorar o sistema para alterar a forma como as mensagens são imprimidas, já estaria tudo centralizado em um lugar.